test/integration_test/azure_disk_test.go

```diff
@@ -0,0 +1,123 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	corev1 "github.com/libopenstorage/operator/pkg/apis/core/v1"
+	"github.com/libopenstorage/operator/pkg/util/test"
+	"github.com/libopenstorage/operator/test/integration_test/types"
+	ci_utils "github.com/libopenstorage/operator/test/integration_test/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+	"time"
+)
+
+var (
+	disksEnc = []string{"type=Premium_LRS,size=100,diskEncryptionSetID=invalid"}
+)
+
+var TestAzureDiskEncryptionCases = []types.TestCase{
+	{
+		TestName:        "InstallWithInvalidDiskEncryptionSetID",
+		TestrailCaseIDs: []string{"C82917"},
+		TestSpec: func(t *testing.T) interface{} {
+			cluster := &corev1.StorageCluster{}
+			cluster.Name = "px-cluster-azure-incorrect-enc"
+			err := ci_utils.ConstructStorageCluster(cluster, ci_utils.PxSpecGenURL, ci_utils.PxSpecImages)
+			require.NoError(t, err)
+
+			cluster.Spec.CloudStorage = &corev1.CloudStorageSpec{
+				CloudStorageCommon: corev1.CloudStorageCommon{
+					DeviceSpecs: &disksEnc,
+				},
+			}
+			cluster.Spec.DeleteStrategy = &corev1.StorageClusterDeleteStrategy{
+				Type: corev1.UninstallAndWipeStorageClusterStrategyType,
+			}
```

Comment:
I believe we don't need this. When we call `UninstallAndValidateStorageCluster()`, it will always set the UninstallAndWipe strategy by default, if empty.